crates/forge/tests/integration/setup_fork.rs

```diff
@@ -160,3 +162,67 @@ fn fork_cairo0_contract() {
 
     assert_passed!(result);
 }
+
+#[test]
+fn get_block_timestamp_in_forked_block() {
+    let test = test_case!(formatdoc!(
+        r#"
+            use result::ResultTrait;
+            use array::ArrayTrait;
+            use option::OptionTrait;
+            use traits::TryInto;
+            use starknet::ContractAddress;
+            use starknet::Felt252TryIntoContractAddress;
+            use starknet::contract_address_const;
+            use snforge_std::{{ BlockTag, BlockId, declare, ContractClassTrait }};
+
+            #[starknet::interface]
+            trait IERC20Camel<TContractState> {{
+                fn return_block_timestamp(ref self: TContractState) -> felt252;
+            }}
+
+            #[starknet::interface]
+            trait IWarpChecker<TContractState> {{
+                fn get_block_timestamp(ref self: TContractState) -> u64;
+            }}
+
+            #[test]
+            #[fork(url: "{}", block_id: BlockId::Number(315057))]
+            fn test_fork_get_block_timestamp_contract_on_testnet() {{
+                let dispatcher = IERC20CamelDispatcher {{
+                    contract_address: contract_address_const::<1825832089891106126806210124294467331434544162488231781791271899226056323189>()
+                }};
+
+                let timestamp = dispatcher.return_block_timestamp();
+                assert(timestamp != 0, timestamp);
```

Comment:
can we assert a real timestamp?

Reply:
Shouldn't we also verify the whole block info instead of just the timestamp?